src/tests/test_process.py

```diff
@@ -94,8 +129,35 @@ def _test_threads(self):
         #   - InvBroadcaster
         #   - AddrBroadcaster
         #   - Downloader
-        self.assertEqual(
-            len(self.process.threads()), self._threads_count)
+        #   - Uploader
+
+        self.longMessage = True
```

Comment:
I cannot find where it's used

Reply:
Oh, it's from unittest.TestCase. Got it.

Reply:
Yes it's so that we have more info about the failures.